test/test_app/config/storage.yml

```diff
@@ -0,0 +1,7 @@
+test:
```

Comment:
Oh hmm. Maybe we should not depend on ActiveStorage? Is it possible to only depend on a few select Rails components instead?